bin/sshtunnel-conf.sh

```diff
@@ -15,6 +15,7 @@ Host bastion
      BatchMode yes
      ExitOnForwardFailure yes
      StrictHostKeyChecking no
+     ServerAliveInterval 60
```

Comment:
I think we want the interval to be lower. Its friend `ServerAliveCountMax` defaults to 3, so a setting of 60 will take 3m to realize a connection is dead.